src/components/Settings/index.tsx

```diff
@@ -72,7 +59,11 @@ function Settings() {
     setOn: setUseGameMode
   } = useToggle(false)
   const { on: showFps, toggle: toggleFps, setOn: setShowFps } = useToggle(false)
-  const { on: offlineMode, toggle: toggleOffline, setOn: setShowOffline } = useToggle(false)
+  const {
```

Comment:
Gonna need standardised style guidelines on this one too....

Reply:
Yeah, the thing is that I'm not sure If the prettier config is working for everyone the same way.
Every time I save a file, something moves. I set up my vscode to format on save, but using the prettier rules we have on the prettierrc file.